src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -13,318 +13,28 @@
 using namespace Microsoft::Console::VirtualTerminal;
 using namespace Microsoft::Console::VirtualTerminal::DispatchTypes;
 
-// Routine Description:
-// - Small helper to disable all color flags within a given font attributes field
-// Arguments:
-// - attr - Font attributes field to adjust
-// - isForeground - True if we're modifying the FOREGROUND colors. False if we're doing BACKGROUND.
-// Return Value:
-// - <none>
-void AdaptDispatch::s_DisableAllColors(WORD& attr, const bool isForeground) noexcept
-{
-    if (isForeground)
-    {
-        WI_ClearAllFlags(attr, FG_ATTRS);
-    }
-    else
-    {
-        WI_ClearAllFlags(attr, BG_ATTRS);
-    }
-}
-
-// Routine Description:
-// - Small helper to help mask off the appropriate foreground/background bits in the colors bitfield.
-// Arguments:
-// - attr - Font attributes field to adjust
-// - applyThis - Color values to apply to the low or high word of the font attributes field.
-// - isForeground - TRUE = foreground color. FALSE = background color.
-//   Specifies which half of the bit field to reset and then apply wApplyThis
-//   upon.
-// Return Value:
-// - <none>
-void AdaptDispatch::s_ApplyColors(WORD& attr, const WORD applyThis, const bool isForeground) noexcept
-{
-#pragma warning(suppress : 26496) // SA is wrong. This variable is assigned more than once.

```

Comment:
I love that this SA comment is just.. entirely incorrect. It's assigned once. LOL.

Reply:
That'd be me. I had the arguments that were const vs. not in `WI_UpdateFlagsInMask` incorrect in my head. Woops.